sample.env

```diff
@@ -18,3 +18,6 @@ VITE_GITHUB_API_TOKEN=
 
 # Snow height page
 VITE_SNOW_HEIGHT_URL="https://www.sneeuwhoogte.nl/sneeuwhoogte-op-je-site/la plagne/big"
+
+# Weather API Key
```

Comment:
```suggestion
# Weather API Key, to be found in our Bitwarden under 'secret_radio'
```

(I suggest we do this everywhere)

Reply:
I copied the key to the IT-Crowd bitwarden as well

Reply:
> (I suggest we do this everywhere)

I agree, although that should be the topic of a new issue / PR. I applied this change for now.